README.md

```diff
@@ -28,8 +28,10 @@ also need to build a docker image.
 
 First you need to get Gerrit to generate an API cookie; you can do this by:
 
-- Going to your Gerrit's webpage
-- Clicking on "Generate Password"
+- Going to your Gerrit's dashboard
```

Comment:
nit: perhaps include an example like "(e.g. https://linux-review.googlesource.com/)" or maybe directly link to https://linux-review.googlesource.com/settings ?

Reply:
Added a direct link.